<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textLog">
    <property name="geometry">
     <rect>
      <x>203</x>
      <y>70</y>
      <width>431</width>
      <height>281</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>390</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSend">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>430</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>전송</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConnect">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>430</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>연결</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leNickname">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>390</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>닉네임 입력</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblVideo">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>80</y>
      <width>321</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string>나의 화면</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWatch">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leTarget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblOpponent">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>151</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1018</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
